_includes/tutorials/console-consumer-read-specific-offsets-partition/kafka/markup/dev/consume-topic-partition-offsets.adoc

```diff
@@ -0,0 +1,23 @@
+So far you've learned how to consume records from a specific partition. But when you specify the partition, you can optionally specify the offest to start conuming from.  Specifying a specific offset can be helpful when debugging an issue, in that you can skip consuming records that you konw aren't a potential problem.
+
+If you haven't done so already, close the previous console consumer with a `CTRL+C`.
+
+From the previous step you know there are 9 records in the second partition.  In this step you'll only consume records starting from offset 5. The changes in this command include removing the `--from-begining` propery and adding an `--offset` flag
+
+It's import to note, when you specify the offset to start consuming from, it's _**exclusive**_ of the number given.  By giving a specific offset, you are saying "I'm only interested in records _**after**_ this specific offset".  That's exactly how commiting works in Kafka, consumer commits the offset of the last record read, so it will start consuming from the _*next*_ available offset.
+
+Here's the command to read records from the second partition starting at offset 6:
+
++++++
+<pre class="snippet"><code class="shell">{% include_raw tutorials/console-consumer-read-specific-offsets-partition/kafka/code/tutorial-steps/dev/console-consumer-keys-partition-offset.sh %}</code></pre>
++++++
+
+After a few seconds you should see something like this
+
++++++
+<pre class="snippet"><code class="shell">{% include_raw tutorials/console-consumer-read-specific-offsets-partition/kafka/code/tutorial-steps/dev/expected-output-step-three.txt %}</code></pre>
++++++
+
+So you can see here, you've consumed records starting from offset 6 to the end, which includes record with offsets of `7, 8, and 9` the last three records.
```

Comment:
Pretty sure this should read 6, 7, 8

```suggestion
So you can see you've consumed records starting from offset 6 to the end, which includes record with offsets of `6, 7, and 8`, the last three records.
```

Here is an equivilant kafkacat command which prints the offsets:
```
kafkacat -b :29092 -t example-topic -C -p 1 -o 6 -f 'Topic %t [%p] at offset %o: key %k: %s\n'
Topic example-topic [1] at offset 6: key key2: Go to
Topic example-topic [1] at offset 7: key key3: Kafka
Topic example-topic [1] at offset 8: key key4: summit
% Reached end of topic example-topic [1] at offset 9
```

Reply:
oof, I confused myself at some point, good catch, I've updated this